package.json

```diff
@@ -80,6 +83,7 @@
     "react-cookie": "^4.1.1",
     "react-dom": "^18.2.0",
     "react-stately": "^3.23.0",
+    "request": "^2.88.2",
```

Comment:
Where is this used? I'm not seeing an import for it ([plus it's a really old, deprecated package](https://www.npmjs.com/package/request))

Reply:
I'm pretty sure this and the `encoding` one below came up while testing the local GCP pubsub emulator, but I will double-check. If that's true (and it's not needed for production) I will move these to dev dependencies, but I'm not totally sure yet.

Reply:
OK re-tested and they don't seem to be needed, was probably from some local debugging rabbit hole, good catch! Removed